msteams-platform/graph-api/rsc/resource-specific-consent.md

```diff
@@ -143,7 +143,7 @@ The following table provides RSC permissions for a chat or meeting and their app
 | `OnlineMeetingRecording.Read.Chat` | Read the recordings of the meetings associated with this chat. |NA |Supported |
 | `OnlineMeetingNotification.Send.Chat` | Send notifications in the meetings associated with this chat. |NA |Supported |
 | `InAppPurchase.Allow.Chat` | Show and complete in-app purchases for users in this chat and any associated meetings. |Supported |NA |
-| `LiveShareSession.ReadWrite.Chat` | Create and synchronize Live Share sessions for meetings associated with this chat. |Supported |NA |
+| `LiveShareSession.ReadWrite.Chat` | Allows the app to create and synchronize Live Share sessions for the chat and provide access related information about the chat's roster, such as name and meeting role, on behalf of the signed-in user. |Supported |NA |
```

Comment:
provide access related information, such as name and role, about the chat's roster and any associated meetings, on behalf of the signed-in user.

Reply:
Updated as suggested